lib/src/main/scala/org/typelevel/scalacoptions/ScalacOptions.scala

```diff
@@ -41,6 +41,9 @@ private[scalacoptions] trait ScalacOptions {
   ): ScalacOption =
     ScalacOption(option, args, isSupported)
 
+  /** Set the encoding to UTF-8 */
+  val encodingUTF8 = encoding("utf8")
```

Comment:
I'm not sure including this option is necessary, We already include utf encoding in default options: https://github.com/typelevel/scalac-options/blob/ed64f8d46ff313aa90f00a691da4264ec93b50c0/lib/src/main/scala/org/typelevel/scalacoptions/ScalacOptions.scala#L888

Reply:
Just because it's widely used with UTF-8